src/app/components/info-card/info-card.component.ts

```diff
@@ -0,0 +1,12 @@
+import { Component, Input } from '@angular/core';
+import { InfoCard } from '../../models/InfoCard.model';
+@Component({
+  selector: 'app-info-card',
+  templateUrl: './info-card.component.html',
+  styleUrls: ['./info-card.component.scss'],
+})
+export class InfoCardComponent {
+  @Input() cards: InfoCard[];
```

Comment:
Non prende in input un array, ma la singola istanza. Questo componente torna una sola card. Modificare anche il template(html)